stage/tests/studio/test_pages.py

```diff
@@ -4,21 +4,22 @@
 from bok_choy.web_app_test import WebAppTest
 
 from edxapp_pages.studio.asset_index import AssetIndexPage
-from edxapp_pages.studio.checklists import ChecklistsPage
-from edxapp_pages.studio.course_import import ImportPage
+#from edxapp_pages.studio.checklists import ChecklistsPage
+#from edxapp_pages.studio.course_import import ImportPage
 from edxapp_pages.studio.course_info import CourseUpdatesPage
 from edxapp_pages.studio.edit_tabs import PagesPage
-from edxapp_pages.studio.export import ExportPage
+#from edxapp_pages.studio.export import ExportPage
 from edxapp_pages.studio.howitworks import HowitworksPage
 from edxapp_pages.studio.index import DashboardPage
 from edxapp_pages.studio.login import LoginPage
-from edxapp_pages.studio.manage_users import CourseTeamPage
+#from edxapp_pages.studio.manage_users import CourseTeamPage
 from edxapp_pages.studio.overview import CourseOutlinePage
 from edxapp_pages.studio.settings import SettingsPage
 from edxapp_pages.studio.settings_advanced import AdvancedSettingsPage
 from edxapp_pages.studio.settings_graders import GradingPage
 from edxapp_pages.studio.signup import SignupPage
 from edxapp_pages.studio.textbooks import TextbooksPage
+import os
```

Comment:
Done. 
This test has to be updated later on. This was the test written by Jay when he made this repo.